evm/src/trace/mod.rs

```diff
@@ -333,11 +338,20 @@ impl fmt::Display for CallTrace {
                 RawOrDecodedCall::Decoded(func, inputs) => (func.clone(), inputs.join(", ")),
             };
 
+            let action = match self.kind {
+                CallKind::Call => "call",
+                CallKind::StaticCall => "staticcall",
+                CallKind::CallCode => "callcode",
+                CallKind::DelegateCall => "delegatecall",
+                _ => unreachable!(),
+            };
+
             let color = trace_color(self);
             write!(
                 f,
-                "[{}] {}::{}{}({})",
+                "[{}] {} {}::{}{}({})",
                 self.gas_cost,
+                Colour::Yellow.paint(action),
```

Comment:
can we make the color different from yellow, since that is what we use for contract creation with "new"?